app/src/main/java/com/xephorium/crystalnote/data/model/CrystalNoteTheme.kt

```diff
@@ -21,6 +21,7 @@ data class CrystalNoteTheme(
 ) {
 
     enum class Themes(val displayName: String, val resourceId: Int) {
+        BLACK("Black", R.style.Black),
```

Comment:
The convention I've been using for sort order has been:
```
1. Base Themes
   a. Light
   b. Dark
2. Child Themes
```
Since this one's a child theme (see parent comment), would you mind bumping this down two lines (to right under `DARK`)?

Reply:
Resolved.